Userland/Libraries/LibCore/Directory.h

```diff
@@ -33,8 +33,8 @@ class Directory {
         Yes,
     };
 
-    static ErrorOr<Directory> create(LexicalPath path, CreateDirectories);
-    static ErrorOr<Directory> create(String path, CreateDirectories);
+    static ErrorOr<Directory> create(LexicalPath path, CreateDirectories, mode_t = 0755);
+    static ErrorOr<Directory> create(String path, CreateDirectories, mode_t = 0755);
```

Comment:
I think the name for the mode should be along the lines of `creation_mode` to clarify that it's only used for creating new directories, not anything else.

Reply:
Yup, that's a much better name! Changed.